src/appleseed/CMakeLists.txt

```diff
@@ -1284,7 +1284,10 @@ set (renderer_modeling_bssrdf_sources
     renderer/modeling/bssrdf/bssrdf.h
     renderer/modeling/bssrdf/bssrdffactoryregistrar.cpp
     renderer/modeling/bssrdf/bssrdffactoryregistrar.h
+    renderer/modeling/bssrdf/bssrdfsample.h
     renderer/modeling/bssrdf/bssrdftraits.h
+    renderer/modeling/bssrdf/dirpolebssrdf.cpp
```

Comment:
Oh ok... In that case I'd strongly prefer to spell it out completely: directionaldipolebssrdf.h... Not *that* long, and so much clearer :)